packages/server/src/games.ts

```diff
@@ -1,5 +1,5 @@
 import { Server as IOServer } from 'socket.io';
-import { last, cloneDeep } from 'lodash';
+import { join, last, cloneDeep } from 'lodash';
```

Comment:
do you use this join?

Reply:
No, I was gonna, but not. I also added `noUnusedParameters` to warn us in the future.